README.md

```diff
@@ -1,6 +1,6 @@
 `pretty_midi` contains utility function/classes for handling MIDI data, so that it's in a format which is easy to modify and extract information from.
 
-Documentation is available [here](http://craffel.github.io/pretty-midi/).  You can also find a Jupyter notebook tutorial [here](https://nbviewer.org/github/craffel/pretty-midi/blob/main/Tutorial.ipynb).
+Documentation is available [here](http://craffel.github.io/pretty-midi/).  You can also find a Jupyter notebook tutorial [here](https://nbviewer.org/github/craffel/pretty-midi/blob/main/Tutorial.ipynb). [[Colab](https://colab.research.google.com/github/craffel/pretty-midi/blob/main/Tutorial.ipynb)]
```

Comment:
Can you make this a little more grammatical, e.g. "You can also find a Jupyter notebook [here] (click [here] to load in Colab)." or something

Reply:
Yeah! Done!